### PR TITLE
Kevin/update ib comission page

### PR DESCRIPTION
**Pull request by kevinw-binary**
IB - comission page
updates :
1. update values on tables
2. add formula
3. add svg (enhanced with svgo)

### CODE REVIEW

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 1]
```diff
@@ -160,72 +160,77 @@ const TypesOfPartnership = () => (
                                     tbody: [
                                         [
                                             { text: it.L('Crash 1000 Index') },
-                                            { text: it.L('0.50') },
+                                            { text: it.L('0.25') },
```
Comment (akmal-binary) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:163]: @negar-binary what is the reason we translated numbers in the first place? 😄

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 2]
```diff
@@ -260,48 +265,92 @@ const TypesOfPartnership = () => (
...
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('For cryptocurrency assets, a deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) will pay out USD 20 in commission based on the following formula:')}</p>
+                                <img
+                                    src={it.url_for('images/pages/affiliates_ib_landing/formulas/formula-financial-crypto.svg')}
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('[_1]Please note[_2]:','<strong>','</strong>')}</p>
```
Comment (akmal-binary) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:351]: We don't need to add strong tag into translation, you can just place it around it.L

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 3]
```diff
@@ -336,54 +385,128 @@ const TypesOfPartnership = () => (
...
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('For cryptocurrency assets, a deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) will pay out USD 20 in commission based on the following formula: ')}</p>
+                                <img
+                                    src={it.url_for('images/pages/affiliates_ib_landing/formulas/formula-financial-stp-crypto.svg')}
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('[_1]Please note[_2]:','<strong>', '</strong>')}</p>
```
Comment (akmal-binary) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:472]: Same here

**Review by akmal-binary — CHANGES_REQUESTED** [review]

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx (hunk 1 above)
Comment (negarn) [reply src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:163]: @akmal-binary the separator in different languages is different. some use comma, some use space

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 4]
```diff
@@ -160,72 +160,77 @@ const TypesOfPartnership = () => (
...
-
+                            <div className='affiliate-ib-calculation'>
+                                <h3 className='text-align-left'><strong>{it.L('How it’s calculated')}</strong></h3>
+                                <p className='text-align-left'>{it.L('A deal for 1 lot of the Volatility 75 Index for a price of USD 500,000 will pay out USD 25 in commission based on the following formula:')}</p>
+                                <img
+                                    src={it.url_for('images/pages/affiliates_ib_landing/formulas/formula-synthetic.svg')}
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('[_1]Please note[_2]: Commission payout for all assets will be converted to your deposit currency based on the latest exchange rate.','<strong>','</strong>')}</p>
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:232]:
```suggestion
                                <p className='text-align-left'>{it.L('[_1]Please note[_2]: Commission payout for all assets will be converted to your deposit currency based on the latest exchange rate.', '<strong>', '</strong>')}</p>
```

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 5]
```diff
@@ -336,54 +385,128 @@ const TypesOfPartnership = () => (
...
                             />
+                            <div className='affiliate-ib-calculation'>
+                                <h3 className='text-align-left'><strong>{it.L('How it’s calculated')}</strong></h3>
+                                <p className='text-align-left'>{it.L('For forex, a deal for 1 lot of EUR/USD will pay out EUR 2.5 in commission based on the following formula:')}</p>
+                                <img
+                                    src={it.url_for('images/pages/affiliates_ib_landing/formulas/formula-financial-stp-forex.svg')}
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('For cryptocurrency assets, a deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) will pay out USD 20 in commission based on the following formula: ')}</p>
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:466]:
```suggestion
                                <p className='text-align-left'>{it.L('For cryptocurrency assets, a deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) will pay out USD 20 in commission based on the following formula:')}</p>
```

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 6]
```diff
@@ -336,54 +385,128 @@ const TypesOfPartnership = () => (
...
+                                <img
+                                    src={it.url_for('images/pages/affiliates_ib_landing/formulas/formula-financial-stp-crypto.svg')}
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('[_1]Please note[_2]:','<strong>', '</strong>')}</p>
+                                <p className='text-align-left'>{it.L('For forex, your commission is represented in the base currency (EUR in the above example)')}</p>
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:473]: should this have a `.` at the end?

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 7]
```diff
@@ -336,54 +385,128 @@ const TypesOfPartnership = () => (
...
+                                <p className='text-align-left'>{it.L('Commission payout for all assets will be converted to your deposit currency based on the latest exchange rate.')}</p>
+                            </div>
+                            <div className='affiliate-ib-calculation affiliate-ib-calculation-minimum'>
+                                <h3 className='text-align-left'><strong>{it.L('Minimum volume requirement')}</strong></h3>
+                                <p className='text-align-left'>{it.L('To receive the minimum commission (0.01 in any currency) allowed by the system, the minimum volume requirement is calculated based on the following formulas: ')}</p>
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:478]:
```suggestion
                                <p className='text-align-left'>{it.L('To receive the minimum commission (0.01 in any currency) allowed by the system, the minimum volume requirement is calculated based on the following formulas:')}</p>
```

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 8]
```diff
@@ -336,54 +385,128 @@ const TypesOfPartnership = () => (
...
+                            
+                                <p className='text-align-left'><strong>{it.L('Example:')}</strong></p>
+                                <p className='text-align-left'>{it.L('A deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) per USD 100,000 turnover will pay out a commission of USD 20. The minimum volume required to receive the minimum commission of USD 0.01 is determined using the following formula:')}</p>
+                                <img
+                                    src={it.url_for('images/pages/affiliates_ib_landing/formulas/min-financial-stp-crypto-value.svg')}
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('For cryptocurrency assets, a deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) will pay out USD 20 in commission based on the following formula: ')}</p>
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:487]:
```suggestion
                                <p className='text-align-left'>{it.L('For cryptocurrency assets, a deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) will pay out USD 20 in commission based on the following formula:')}</p>
```

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx (hunk 1 above)
Comment (akmal-binary) [reply src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:163]: Oh wow, good to know, I thought some countries actually use different number system 😄

**Review by akmal-binary — DISMISSED** [review]

**Review by negarn — DISMISSED** [review]

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 9]
```diff
@@ -250,74 +257,135 @@ const TypesOfPartnership = () => (
...
-                                            { text: it.L('Commission per round trade') },
+                                            { text: it.L('Asset') },
+                                            { text: it.L('Commission per USD 100K turnover'), className: 'center-text' },
+                                        ],
+                                    ],
+                                    tbody: [
+                                        [
+                                            { text: it.L('Stock indices') },
+                                            { text: it.L('USD 1') },
+                                        ],                                        [
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:268]: why is the opening bracket so far away 😂

File: src/templates/static/affiliate_ib_landing/type_of_partnership.jsx [hunk 10]
```diff
@@ -146,102 +146,109 @@ const TypesOfPartnership = () => (
...
+                                            { text: it.L('(1 standard forex lot is 100k units)'), className: 'center-text' },
                                         ],
                                     ],
                                     tbody: [
                                         [
-                                            { text: it.L('Forex and metals*') },
-                                            { text: it.L('10') },
+                                            { text: it.L('Forex') },
+                                            { text: it.L('5') },
+                                        ],                                        [
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing/type_of_partnership.jsx:249]: same here

**Review by negarn — APPROVED** [review]

**Pull request merged**